docs/usage/quickstart.rst

```diff
@@ -5,33 +5,92 @@
+to installation procedures, the only significant difference between a test
+cluster and a full production environment is the amount of resources required
+to implement the design.
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
 .. _CentOS: https://www.centos.org
 .. _Kubernetes: https://kubernetes.io
 
-Defining an Inventory
----------------------
-To tell the Ansible_-based deployment system on which machines MetalK8s should
-be installed, a so-called *inventory* needs to be provided. This inventory
-contains a file listing all the hosts comprising the cluster, as well as some
-configuration.
+General Cluster Requirements
+----------------------------
+Setting up a MetalK8s test cluster quickly requires three machines
+running CentOS_ 7.4 or higher (these can be VMs) to which you have SSH access.
+Each machine acting as a Kubernetes_ node (all three in the present example)
+must also have at least one disk available to provision storage volumes.
+
+Sizing
+^^^^^^
+
+Each node must satisfy the following sizing requirements.
+
+.. note::
+   The root file system requires at least 20 GB.
+
++-----------------+--------+--------+-------+
+|    Component    | etcd   | Master | Node  |
++=================+========+========+=======+
+| Cores           | 2      | 4      | 4     |
++-----------------+--------+--------+-------+
+| RAM             | 4 GB   | 8 GB   | 8 GB  |
++-----------------+--------+--------+-------+
+| Minimum         |        |        |       |
+| dedicated       |        |        |       |
+| storage capacity|        |        |       |
+| required        |    -   |    -   | 128 GB|
++-----------------+--------+--------+-------+
+
+Proxies
+^^^^^^^
+For nodes operating behind a proxy, add the following lines to each cluster
+server's :file:`/etc/environment` file:
+
+.. code-block:: shell
+
+ http_proxy=http://user;pass@<HTTP proxy IP address>:<port>
+ https_proxy=http://user;pass@<HTTPS proxy IP address>:<port>
+ no_proxy=localhost,127.0.0.1,10.*
+
+Get the MetalK8s Git Repo
+-------------------------
+Go to the MetalK8s releases_ page and download the source code zip file
+for the release you wish to get.
+
+Define an Inventory
+-------------------
+Each server must be configured in an inventory that identifies the servers to
+the Ansible_-based deployment system, as well as their basic configuration,
+including masters and nodes.
+
+The inventory is a directory that contains a :file:`hosts` file, which lists
+all hosts in the cluster, and a subdirectory (:file:`group_vars`) that contains
+:file:`kube-node.yml`, a configuration file.
 
 .. _Ansible: https://www.ansible.com
 
-First, create a directory, e.g. :file:`inventory/quickstart-cluster`, in which
-the inventory will be stored. For our setup, we need to create two files. One
-listing all the hosts, aptly called :file:`hosts`:
+To create an inventory:
+
+1. Log in to the machine to which you cloned or put the git repo, as described
+   in the previous section.
```

Comment:
The Git repository is not cloned in the previous section (anymore).

Reply:
Can we talk about a "MetalK8s project"? or "release"?